cpp/include/cudf_test/cudf_gtest.hpp

```diff
@@ -114,8 +114,14 @@ struct TypeList<Types<TYPES...>> {
       exception);                                                   \
   } while (0)
 
-#define CUDF_EXPECT_THROW_MESSAGE(x, msg) \
-  EXPECT_THROW_MESSAGE(x, cudf::logic_error, "cuDF failure at:", msg)
+#define CUDF_EXPECT_THROW_MESSAGE(...)                                     \
```

Comment:
I don't know when these test macros snuck in, but I recall a long time ago advocating strongly against them. I think it's fragile and unimportant for us to be testing for the exact contents of an exceptions error message.

Don't block this PR on removing these, but file it away in your memory banks for future work.